app_dart/bin/gae_server.dart

```diff
@@ -23,12 +23,15 @@ Future<void> main() async {
       accessTokenService: AccessTokenService.defaultProvider(config),
     );
 
+    final fusionTester = FusionTester();
+
     /// LUCI service class to communicate with buildBucket service.
     final LuciBuildService luciBuildService = LuciBuildService(
       config: config,
       cache: cache,
       buildBucketClient: buildBucketClient,
       pubsub: const PubSub(),
+      fusionTester: fusionTester,
```

Comment:
any reason not to just invoke the constructor here on this line?

Reply:
Its a shared caching service; the Scheduler uses it as well.